peps/pep-0750.rst

```diff
@@ -399,14 +458,21 @@ The debug specifier, ``=``, is supported in template strings and behaves similar
 to how it behaves in f-strings, though due to limitations of the implementation
 there is a slight difference.
 
-In particular, ``t'{expr=}'`` is treated as ``t'expr={expr}'``:
+In particular, ``t'{expr=}'`` is treated as ``t'expr={expr!r}'``:
 
 .. code-block:: python
 
     name = "World"
     template = t"Hello {name=}"
     assert template.args[0] == "Hello name="
     assert template.args[1].value == "World"
+    assert template.args[1].conv == "r"
+
+If a separate format string is also provided, ``t'{expr=:fmt}`` is treated instead as
+``t'expr={expr!s:fmt}'``.
+
+Whitespace is preserved in the debug specifier, so ``t'{expr =}'`` is treated as
+``t'expr ={expr!r}'``.
```

Comment:
Maybe use `expr = ` in the example to clarify that whitespace before _and_ after the `=` is preserved.